example/eblob_backend.cpp

```diff
@@ -1528,3 +1538,48 @@ int blob_send_new(struct eblob_backend_config *c, void *state, struct dnet_cmd *
 	         dnet_print_error(err));
 	return err;
 }
+
+int blob_bulk_read_new(struct eblob_backend_config *c, void *state, struct dnet_cmd *cmd, void *data) {
+	using namespace ioremap::elliptics;
+
+	if (c == nullptr || state == nullptr || cmd == nullptr || data == nullptr)
+		return -EINVAL;
+
+	auto st = reinterpret_cast<dnet_net_state *>(state);
+	const int backend_id = c->data.stat_id;
+	auto backend_io = dnet_get_backend_io(st->n->io, backend_id);
+	if (!backend_io) {
+		DNET_LOG_ERROR(c->blog, "EBLOB: {}: couldn't find backend_io for backend_id: {}",
+			       __func__, backend_id);
+		return -EINVAL;
+	}
+
+	dnet_bulk_read_request bulk_request;
+	deserialize(data_pointer::from_raw(data, cmd->size), bulk_request);
+
+	int err = 0;
+	dnet_read_request request;
+	request.ioflags = bulk_request.ioflags;
+	request.read_flags = bulk_request.read_flags;
+	request.data_offset = request.data_size = 0;
+	dnet_empty_time(&request.deadline);
+
+	cmd->flags &= ~DNET_FLAGS_NEED_ACK;
+	const auto num_keys = bulk_request.keys.size();
+	for (size_t i = 0; i < num_keys; ++i) {
+		struct dnet_cmd cmd_copy(*cmd);
```

Comment:
cmd_copy is being modified by dnet_send_reply/blob_read_new_impl. That's why I've left it inside a loop. Ok?